pkgs/tools/misc/wacomtablet-kcm/default.nix

```
@@ -0,0 +1,32 @@
+{
+  stdenv, mkDerivation, fetchurl, extra-cmake-modules, qtx11extras,
+  plasma-workspace, libwacom, xf86_input_wacom
+}:
+
+mkDerivation {
+  name = "wacomtablet-kcm";
+  version = "3.2.0";
+  src = fetchurl {
+    url = "mirror://kde/stable/wacomtablet/3.2.0/wacomtablet-3.2.0.tar.xz";
```

```suggestion
    url = "mirror://kde/stable/${pname}/{$version}/${pname}-${version}.tar.xz";
```

(needs `mkDerivation rec {`)

---

This doesn't currently work as `name`/`pname` is `wacomtablet-kcm`, while the upstream source is named `wacomtablet`. I appended "kcm" to the package name as I thought it would be clearer in the context of nixpkgs, but I could rename it "wacomtablet" to match upstream.

---

What does "kcm" mean? You can still use version

---

KDE Configuration Module. Each part of the KDE system settings is a kcm. Most of them are bundled in the core plasma5 packages, but there are a few individually packaged ones such as `sddm-kcm` and now this one. There are also a couple of other packages which include a kcm alongside the thing which it configures.

---

I'm thinking I'll probably drop the `-kcm` and name it `wacomtablet` like upstream. It should be clear enough from the description that it's the KDE config GUI.

---

I've changed the package name to `wacomtablet` and used `pname` and `version` in the url. Thanks!